src/randoop/operation/ConstructorSignatures.java

```diff
@@ -0,0 +1,73 @@
+package randoop.operation;
+
+import java.lang.reflect.Constructor;
+
+/**
+ * ConstructorParser provides static methods to write as well as parse 
+ * a string representation of a constructor signature.
+ */
+public class ConstructorSignatures {
+
+  /**
+   * Parses a constructor signature as produced by 
+   * {@link ConstructorSignatures#getSignatureString(Constructor)} 
+   * and returns the corresponding reflective {@link Constructor} object.
+   * 
+   * @param signature a string representing a constructor signature.
+   * @return reflective {@link Constructor} method corresponding to signature.
+   * @throws OperationParseException if signature parameter does not match 
+   *         expected format.
+   */
+  public static Constructor<?> getConstructorForSignatureString(String signature) throws OperationParseException {
```

Comment:
Most of this implementation is identically duplicated in getMethodForSignatureString, and it would be good to eliminate that duplication.